src/pages/Intents/IntentIdentifiers.js

```diff
@@ -0,0 +1,135 @@
+import i18n from '@dhis2/d2-i18n'
+import PropTypes from 'prop-types'
+import React, { useState } from 'react'
+import { MultiSelectField } from '../../components/field/MultiSelectField'
+import { SelectField } from '../../components/field/SelectField'
+import { TextField } from '../../components/field/TextField'
+
+const screenActionOptions = [
+    { label: 'Search', value: 'SEARCH' },
+    { label: 'Data Entry', value: 'DATA_ENTRY' },
+]
+
+const elementTypeOptions = [
+    {
+        value: 'dataElements',
+        label: i18n.t('Data Element'),
+    },
+    {
+        value: 'trackedEntityAttributes',
+        label: i18n.t('Tracked Entity Attribute'),
+    },
+]
+
+const IntentIdentifiers = ({
+    formData,
+    onChange,
+    dataElements,
+    attributes,
+}) => {
+    const [selectedElementType, setSelectedElementType] = useState('')
+    const [list, setList] = useState([])
```

Comment:
I don't think you need to keep these in state. You could just conditionally use dataElements or attributes based on the value of `selectedElementType`?